app/gui/src/project-view/stores/suggestionDatabase/documentation.ts

```diff
@@ -79,6 +82,7 @@ export function documentationData(
         .split(/\s*,\s*/g) ?? [],
     isPrivate: isSome(tagValue(parsed, 'Private')),
     isUnstable: isSome(tagValue(parsed, 'Unstable')) || isSome(tagValue(parsed, 'Advanced')),
+    suggestedRank: suggestedRank != null ? parseFloat(suggestedRank) : undefined,
```

Comment:
If `parseFloat` returns `NaN`, we might convert that to `undefined`.

Test data could also include `SUGGESTION` tag with invalid or missing arguments.